### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by machacekondra**
## Problem
<Please describe what problem this PR is trying to resolve>
This PR add execution of `ansible-lint` to each PR using github actions. Also this PR solves all `ansible-lint` issues, except the one specified in `.ansible-lint` file in the root directory.

## Solution
<Please elaborate the solution for the problem>

## Tests
<Please provide steps to test the PR>

## Notes
<Additional comments for the PR>

### CODE REVIEW

**Comment by machacekondra** [comment]
Verification in progress: https://github.com/machacekondra/sap-hana/pull/2/checks?check_run_id=3636887413

**Review by KimForss — COMMENTED** [review]
The whitespace is intentional to improve readability.

File: .github/workflows/github-actions-ansible-lint.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: BSFishy/pip-action@v1
+      with:
+        packages: |
+          ansible==2.9
+          ansible-lint
+    - run: "ansible-lint -x role-name,line-length,comments,comments-indentation,empty-string-compare -v --force-color ${{ github.workspace }}/deploy/ansible/*role* --exclude=${{ github.workspace }}/deploy/ansible/XXroles --exclude=${{ github.workspace }}/deploy/ansible/vars/"
```
Comment (KimForss) [inline .github/workflows/github-actions-ansible-lint.yml:13]: can we add colons, commas and braces to be excluded - that way we keep the readability as we want it to be

**Comment by machacekondra** [comment]
> The whitespace is intentional to improve readability.

Makes sense. Would be nice to keep it in all files. So I will add ignore for that check.

File: .github/workflows/github-actions-ansible-lint.yml (hunk 1 above)
Comment (KimForss) [reply .github/workflows/github-actions-ansible-lint.yml:13]: I ignore these   
- too many spaces after colon
- too many spaces after comma

After seing your PR I have been running ansible-lint manually to fix issues

**Review by KimForss — APPROVED** [review]
Looks good to me

**Pull request merged**